en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md

```diff
@@ -2002,7 +1999,9 @@ Follow the instructions below to move all the existing API Manager configuration
 6.  Upgrade the Identity component in WSO2 API Manager from version 5.2.0 to 5.10.0.
 
     !!! note
-        As WSO2 API-M shares identity components with WSO2 Identity Sever (WSO2 IS), this step is necessary to upgrade those components (even if you are not using WSO2 IS as a Key Manager).
+        If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.10.0](../upgrading-wso2-is-as-key-manager/upgrading-from-is-km-520-to-5100.md).
+
+        Even if you are not using WSO2 IS as a Key Manager, following steps are necessary to be followed in order to upgrade identity components as WSO2 API-M shares the identity components with WSO2 Identity Sever (WSO2 IS).
 
```

Comment:
"Even" is not needed right? "if you are not using WSO2 IS as a Key Manager" then you must flow the below setps.

Reply:
+1